apps/test/unit/templates/curriculumCatalog/CurriculumCatalogTest.jsx

```diff
@@ -530,5 +530,22 @@ describe('CurriculumCatalog', () => {
       assert(!replacedLocation.includes('lesson'));
       assert(replacedLocation.includes('duration=week'));
     });
+
+    it('params update when translated toggle is flipped', () => {
```

Comment:
Love this test! I notice you have other tests that test if the filter actually worked (e.g. filtering by grade limits options). Is there a reason that this toggle isn't tested in the same way? I think it'd be great to add that if possible

Reply:
Good call, will do!